training/distributed_training/pytorch/model_parallel/gpt-j/args.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+
+from transformers import TrainingArguments, CONFIG_MAPPING, MODEL_FOR_CAUSAL_LM_MAPPING
+
+from dataclasses import dataclass, field
+from typing import Optional
+
+MODEL_CONFIG_CLASSES = list(MODEL_FOR_CAUSAL_LM_MAPPING.keys())
+MODEL_TYPES = tuple(conf.model_type for conf in MODEL_CONFIG_CLASSES)
+
+
+@dataclass
+class CustomTrainingArguments(TrainingArguments):
+
+    output_dir: Optional[str] = field(
+        default="./temp",
+        metadata={"help": "Output directory"},
+    )
+    same_seed: Optional[int] = field(
+        default=0,
+        metadata={"help": "..."},
+    )
+    beta1: float = field(
+        default=0.9,
+        metadata={"help": "beta1 parameter for Adam optimizer)."},
+    )
+    beta2: float = field(
+        default=0.95,
+        metadata={"help": "beta2 parameter for Adam optimizer)."},
+    )
+    lr: float = field(
+        default=None,
+        metadata={"help": "Initial learning rate"},
+    )
+    weight_decay: Optional[float] = field(
+        default=0.01,
+        metadata={"help": "Weight decay"},
+    )
+    grad_clip: float = field(
+        default=1.0,
+        metadata={"help": "Gradient clipping"},
+    )
+    use_adamw: int = field(
+        default=0,
+        metadata={"help": "Use adamw optimizer."},
+    )
+    lr_decay_iters: int = field(
+        default=None,
+        metadata={
+            "help": "number of iterations to decay learning rate over. If None defaults to train iters."
+        },
+    )
+    max_steps: int = field(
+        default=5000,
+        metadata={"help": "Max steps."},
+    )
+    min_lr: float = field(
+        default=0.0,
+        metadata={
+            "help": "Minumum value for learning rate. The scheduler"
+            "clip values below this threshold."
```

Comment:
Formatting nit - make sure all multi-line strings end with a space at the end of each line (multiple occurrences across py files). 

Grammar/typos: revise to "the scheduler clips" instead of "clip", "Minumum" -> "Minimum"